### DISCUSSION TITLE
Server connections interrupted

### DISCUSSION THREAD

**Discussion by sarah-lital** [iterate-ch/cyberduck#15735] [Q&A]
The attached pop up regularly appears and automatically disappears from my screen. Can be every few second to few minutes. These are specifically the connections I have open.
It stops when I shut down mountain duck. 
Im on MacBook Pro 2021 M1 Sonoma 14.2.1

<img width="271" alt="Screenshot 2024-03-14 at 15 14 21" src="https://github.com/iterate-ch/cyberduck/assets/77614779/83e07780-0d4c-4aa0-bcb7-f2d8b9e7cd36">

**Comment by dkocher** [comment]
This should be resolved in version 4.14.4 and later.

**Reply by sarah-lital** [reply]
Thanks for the reply. I am running version 4.16.1 and it was not resolved. Additionally the sync to my remote connections is very slow and frequently gets stuck since the latest update.

**Reply by dkocher** [reply]
We had another fix in version _[4.17.0](https://mountainduck.io/changelog/)_. Let us know if you can still reproduce and if there is a pattern.

**Comment by tryonkus** [comment]
I am also running 4.16.3 on Sonoma 14.6.1 and getting multiple messages per hour. If I ignore the error it usually has no effect, but sometimes a drive will freeze up, and I then generally have to force quit Mountain Duck.

I am connecting to Google Drive, OneDrive, and Dropbox.

**Reply by dkocher** [reply]
We had another fix in version _[4.17.0](https://mountainduck.io/changelog/)_. Let us know if you can still reproduce and if there is a pattern.

**Reply by tryonkus** [reply]
Installed 4.17.0 and will let you know results.

**Comment by Babbzzz** [comment]
I noticed the issues is a known issue, and possibly not an issue with Mountain Duck. But I am on Sonoma 15.2 and Mountain Duck 4.17.0

**Reply by dkocher** [reply]
We had another fix in version _[4.17.0](https://mountainduck.io/changelog/)_ that should ensure the _Server connections interrupted_ prompt is not shown in Finder. Let us know if you can still reproduce and if there is a pattern.

**Reply by Babbzzz** [reply]
I am using 4.17.1. Issue occurs less now. But still appears.

**Comment by kianrafiee** [comment]
Issue still persists for me on latest version. I see in the documentation this is listed as an Operating System specific issue. Does that mean this will never be resolved?

**Comment by artginzburg** [comment]
Same thing here, Mountain Duck v4.17.1, macOS Sequoia 15.3.1, MBP 2021 M1 Pro.

I'd like to just hide this popup forever (e.g. for a specific connection), but can't find any settings regarding that.

I don't experience any actual problems with the connection, but the popup still appears about 5 times a day.
The connection it warns about is an SFTP. I use Connect Mode: Smart Synchronization.


<details>
<summary> (details)

Here are my defaults (`defaults read io.mountainduck`):
</summary>

```sh
{
    "NSStatusItem Preferred Position Item-0" = 497;
    "NSTableView Columns v3 Versions" = {length = 624, bytes = 0x62706c69 73743030 d4010203 04050607 ... 00000000 000001ac };
    "NSTableView Sort Ordering v2 Versions" = {length = 237, bytes = 0x62706c69 73743030 d4010203 04050607 ... 00000000 000000b5 };
    "NSTableView Supports v2 Versions" = 1;
    "NSWindow Frame Bookmark" = "538 343 435 486 0 0 1512 944 ";
    "NSWindow Frame Info" = "559 456 393 355 0 0 1512 944 ";
    "NSWindow Frame Login" = "546 487 420 294 0 0 1512 944 ";
    "NSWindow Frame Preferences" = "456 129 600 656 0 0 1512 944 ";
    "NSWindow Frame SUUpdateAlert" = "446 409 620 398 0 0 1512 944 ";
    SUAutomaticallyUpdate = 0;
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    SULastCheckTime = "2025-04-26 11:05:43 +0000";
    SUSendProfileInfo = 0;
    SUSkippedVersion = 22638;
    SUUpdateGroupIdentifier = 3137878999;
    "bookmark.4ca091af-e668-437a-81a9-287a218d509d" = 1;
    "bookmark.c5ea6262-bd01-4e8c-b591-251c74c1dcea" = 2;
    "bookmark.d70d5a6c-6c6a-4ef7-b0e6-7b42340ff4bf" = 0;
    "bookmark.import.com.eltima.cloudmounter" = true;
    "bookmark.import.com.expandrive.ExpanDrive" = true;
    "bookmark.import.com.expandrive.ExpanDrive3" = true;
    "bookmark.import.com.expandrive.ExpanDrive34" = true;
    "bookmark.import.com.expandrive.ExpanDrive35" = true;
    "bookmark.import.com.expandrive.ExpanDrive36" = true;
    "bookmark.import.com.panic.Transmit" = true;
    "bookmark.import.com.panic.Transmit5" = true;
    "connection.mount.notification.enable" = false;
    "connection.port.default" = 22;
    "connection.protocol.default" = "sftp-iterate GmbH";
    "connection.unmount.notification.enable" = false;
    "fs.lock.enable" = true;
    "fs.sync.notification.conflict" = false;
    "fs.sync.notification.download" = false;
    "fs.sync.notification.error" = false;
    "fs.sync.notification.placeholder" = false;
    "fs.sync.queue.paused" = true;
    "info.toolbar.selected" = 0;
    "preferences.toolbar.selected" = 1;
    "session.d70d5a6c-6c6a-4ef7-b0e6-7b42340ff4bf" = 0;
    "update.check" = false;
    "update.check.guid" = "8597cc88-1a94-46af-8cb0-2dcd9d1776ae";
}
```
</details>

**Reply by dkocher** [reply]
We had another fix in version [4.17.4](https://mountainduck.io/changelog/). Let us know if you can still reproduce and if there is a pattern.

**Comment by e1eg11t** [comment]
I'm running v4.17.4 on Windows 11 but still having connection (with Cryptomator integration) dropped constantly, which renders the program unusable.  This used not to be a problem for many years.  Could someone please guide me on how to resolve this?

If it makes a difference, the connection is to Google Drive.  When the connection is dropped, all the decrypted folders will disappear from the File Explorer altogether and the message "This folder is empty" would be displayed.  The only way to restore them would be to click on the top folder again on the left panel again.

Many thanks!

Edit: Clarify type of profile used for the connection.

**Reply by dkocher** [reply]
We recommend to use _Smart Synchronization_ connect mode that allows to browse folders while offline.

**Comment by e1eg11t** [comment]
Hi, David, 
It is exactly 4.17.4 I'm on.  Used not to be a problem before my update to this latest version along with the upgrade from Windows 10 to 11.
Thank you again and will wait to hear from you. 

Yahoo Mail: Sent from my mobile device 
 
  On Sat, May 10, 2025 at 4:53, David ***@***.***> wrote:    We had another fix in version 4.17.4. Let us know if you can still reproduce and if there is a pattern. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because 
|   
|   
|    
|  DuckDuckGo removed one tracker.  More   |

   
|  
|   Deactivate   |

  |

    |

   |

   |




We had another fix in version 4.17.4. Let us know if you can still reproduce and if there is a pattern.
